src/renate/benchmark/models/base.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from abc import ABC
+from typing import List, Optional
+
+import torch
+from torch import nn
+
+from renate import defaults
+from renate.models import RenateModule
+from renate.models.prediction_strategies import ICaRLClassificationStrategy, PredictionStrategy
+
+
+# TODO: merge unit tests for the submodules
+class RenateBenchmarkingModule(RenateModule, ABC):
```

Comment:
It was meant to share code between our models but since we decided to move the iCaRL logic to the RenateModule directly, it features something more right now. Will add more documentation.